app/mailers/startup_mailer.rb

```diff
@@ -1,10 +1,12 @@
 # Mails sent out to teams, as a whole.
 class StartupMailer < SchoolMailer
-  def feedback_as_email(startup_feedback)
+  def feedback_as_email(startup_feedback, grading_details)
```

Comment:
Update the second argument to a boolean flag. Do the grading details hash computation here as a private method if `include_grading` is true.